pkg/server/proto/server.proto

```diff
@@ -5420,6 +5472,264 @@ message ProjectTemplate {
   }
 }
 
+message TerraformNocodeModule {
+  // The module name for the Terraform no-code module.
```

Comment:
Is the module name unique, meaning does the "name" include the source repo like `hashicorp/somenocode`? It may be worth noting the expected format here if it's supposed to contain a full repo/name format.

Reply:
Since the Terraform module source format can take many forms, I've updated this comment since the details of the format are there for each source type.

```
  // The source of the Terraform no-code module.
  // https://developer.hashicorp.com/terraform/language/modules/sources
```

Reply:
Makes sense 👍